src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/AnnotationLocationCheck.java

```diff
@@ -95,13 +95,28 @@
  * </li>
  * </ul>
  * <p>
- * Example to allow multiple annotations on the same line
+ * Default configuration:
  * </p>
  * <pre>
- * &#64;SuppressWarnings("deprecation") &#64;Mock DataLoader loader; // no violations
+ * &lt;module name=&quot;AnnotationLocation&quot;/&gt;
  * </pre>
  * <p>
- * Use the following configuration:
+ * Example for default configuration:
+ * </p>
+ * <pre>
+ * &#64;NotNull private boolean field1; //ok
+ * &#64;Override public int hashCode() { return 1; } //ok
+ * &#64;NotNull //ok
+ * private boolean field2;
+ * &#64;Override //ok
+ * public boolean equals(Object obj) { return true; }
+ * &#64;Mock DataLoader loader; //ok
+ * &#64;SuppressWarnings("deprecation") DataLoader loader; //violation
+ * &#64;SuppressWarnings("deprecation") public int foo() { return 1; } //violation
+ * &#64;SuppressWarnings("deprecation") &#64;Mock DataLoader loader; //violation
```

Comment:
this line should show how 2 annations withtout parameter result in violation.

Reply:
Done